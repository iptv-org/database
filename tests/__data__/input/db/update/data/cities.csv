country,subdivision,code,name,wikidata_id
AE,AE-AJ,AEAJM,Ajman,Q530171
AL,AL-08,ALMIL,Milot,Q18749