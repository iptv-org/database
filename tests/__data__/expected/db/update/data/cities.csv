country,subdivision,code,name,wikidata_id
AF,AF-HER,AEAJM,Herāt,Q45313
CN,CN-SD,CNYAT,Yantai,Q210493